nixos/tests/systemd-nspawn-options.nix

```diff
@@ -0,0 +1,48 @@
+# Test for NixOS container options.
+
+import ./make-test.nix ({ pkgs, ...} : {
```

Comment:
We're actually switching our test suite to python-based tests (this can be used with `./make-test-python.nix`).